### PR TITLE
doc: Updated java-microprofile stack readme

### PR DESCRIPTION
**Pull request by neeraj-laad**
Overall re-write of the readme

### CODE REVIEW

File: incubator/java-microprofile/README.md [hunk 1]
```diff
@@ -1,47 +1,63 @@
-# Java Microprofile
+# Java Microprofile Stack
 
-A stack is the foundation of your application which provides a prototype platform definition for Java Microprifile based projects.
+The Java Microprofile stack provides a consistent way of developing microservices based upon the [Eclipse MicroProfile® specifications](https://microprofile.io). This stack lets you use [Maven](https://maven.apache.org) to develop applications for [Open Liberty](https://openliberty.io) runtime, which includes OpenJDK with container-optimizations in OpenJ9.
 
-The Java Microprofile stack uses a parent POM to manage dependency versions and provide required capabilities and plugins whilst using Maven. Specifically, this stack enables adoptopenjdk/openjdk8-openj9 and Open Liberty version `19.0.0.5`.
+The Java Microprofile stack uses a parent Maven project object model (POM) to manage dependency versions and provide required capabilities and plugins.
+
+This stack is based on `adoptopenjdk/openjdk8-openj9` and `Open Liberty v19.0.0.5`. It provides live reloading during development by utilizing `loose application` capabilities.
```
Comment (helenmasters) [inline incubator/java-microprofile/README.md]: what do we mean by `adoptopenjdk/openjdk8-openj9`?

File: incubator/java-microprofile/README.md [hunk 2]
```diff
@@ -1,47 +1,63 @@
-# Java Microprofile
+# Java Microprofile Stack
 
-A stack is the foundation of your application which provides a prototype platform definition for Java Microprifile based projects.
+The Java Microprofile stack provides a consistent way of developing microservices based upon the [Eclipse MicroProfile® specifications](https://microprofile.io). This stack lets you use [Maven](https://maven.apache.org) to develop applications for [Open Liberty](https://openliberty.io) runtime, which includes OpenJDK with container-optimizations in OpenJ9.
```
Comment (helenmasters) [inline incubator/java-microprofile/README.md]: ...which includes OpenJDK makes it sound like that is installed as part of Open Liberty is that the intent?

File: incubator/java-microprofile/README.md [hunk 3]
```diff
@@ -1,47 +1,63 @@
...
 
 ## Templates
 
-Templates are used to create your local project and start your developement. When initializing your project you will be provided with a open liberty sample application by default.
+Templates are used to create your local project and start your development. When initializing your project you will be provided with an Open Liberty template application.
+
+The default template provides a `pom.xml` file that references the parent POM defined by the stack and enables Liberty features that support [Eclipse MicroProfile 2.2](https://openliberty.io/docs/ref/feature/#microProfile-2.2.html). Specifically, this template includes:
 
-The following capabilities are available from the Java Microprofile templates:
+### MicroProfile Health
```
Comment (helenmasters) [inline incubator/java-microprofile/README.md]: Do we need the word Microprofile in front of the titles Health, Metrics and OpenAPI, as I understand e.g. what OpenAPI is but don't get what we mean by "Microprofile OpenAPI"

File: incubator/java-microprofile/README.md (hunk 1 above)
Comment (neeraj-laad) [reply incubator/java-microprofile/README.md]: We use OpenJDK v8 binaries built by AdoptOpenJDK with open j9 optimisations..  I was trying to make that clear.

File: incubator/java-microprofile/README.md (hunk 2 above)
Comment (neeraj-laad) [reply incubator/java-microprofile/README.md]: might be best to say `running on`

File: incubator/java-microprofile/README.md (hunk 3 above)
Comment (neeraj-laad) [reply incubator/java-microprofile/README.md]: MicroProfile have specific features like `MicroProfile Health` or `mpHealth` that provide capabilities for health checking.

File: incubator/java-microprofile/README.md (hunk 2 above)
Comment (neeraj-laad) [reply incubator/java-microprofile/README.md]: addressed

File: incubator/java-microprofile/README.md (hunk 1 above)
Comment (neeraj-laad) [reply incubator/java-microprofile/README.md]: replaced with the English equivalent

File: incubator/java-microprofile/README.md (hunk 3 above)
Comment (neeraj-laad) [reply incubator/java-microprofile/README.md]: removed to keep it less wordy and simple

File: incubator/java-microprofile/README.md [hunk 4]
```diff
@@ -1,47 +1,63 @@
...
+Health endpoint: http://localhost:9080/health
 
-Health Check: http://localhost:9080/health
+### Metrics
 
-### Application metrics
+The `mpMetrics` feature enables MicroProfile Metrics support in Open Liberty. Note that this feature requires SSL and the configuration has been provided for you. You can monitor metrics to determine the performance and health of a service. You can also use them to pinpoint issues, collect data for capacity planning, or to decide when to scale a service to run with more or fewer resources.
 
-Enable powerful monitoring for your distributed application. You can monitor metrics to determine the performance and health of a service. You can also use them to pinpoint issues, collect data for capacity planning, or to decide when to scale a service to run with more or fewer resources.
+Metrics endpoint: http://localhost:9443/metrics
```
Comment (mhamwala) [inline incubator/java-microprofile/README.md]: update the port here

File: incubator/java-microprofile/README.md [hunk 5]
```diff
@@ -1,47 +1,63 @@
...
-2. After your project has been intialized you can then run your application using the following command:
-```bash
-appsody run
-```
-This launches a Docker container that continuously re-builds and re-runs your project, exposing it on port 9080.
+1. You should be able to access the following endpoints, as they are exposed by your template application by default:
 
-Note: you can continue to edit the application in Eclipse or VSCode IDE. Changes will be reflected in the running container around 2 seconds after the changes are saved.
+    - Health endpoint: http://localhost:9080/health
+    - Metrics endpoint: http://localhost:9443/metrics (login as `admin` user with `adminpwd` password)
```
Comment (mhamwala) [inline incubator/java-microprofile/README.md]: Once more here

**Pull request merged**